src/assets/styles/_mixins.scss

```diff
@@ -0,0 +1,104 @@
+$spacing: (
```

Comment:
question: why recreate these utility classes (whole file)?

Reply:
Because of https://github.com/kumahq/kuma-gui/pull/1314#discussion_r1299709412. I want to keep using them in the absence of them in future Kongponents versions.